### ISSUE TITLE
Replace configuration "active" by "instrument"

### ISSUE THREAD

**Issue by SimonSimCity** [kschingiz/meteor-elastic-apm#37]
The version 3.0 of the package `apm-agent-nodejs` introduced a new configuration called `instrument` which has the potential to replace our custom configuration: https://github.com/elastic/apm-agent-nodejs/pull/1114

**Comment by kschingiz** [comment]
Yes, we should try it. I used it a little bit to develop own minor apm services. I think it should be separate repository, then we can compare 2 approaches of instrumenting.
